### DISCUSSION TITLE
What's the expectation around Dependabot pull requests?

### DISCUSSION THREAD

**Discussion by discentem** [renovatebot/renovate#27528] [Request Help] [answered]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### If you're self-hosting Renovate, select which platform you are using.

None

### What is your question?

If I understand correctly I am supposed to keep Dependabot enabled for https://docs.renovatebot.com/configuration-options/#vulnerabilityalerts. But does this mean that Renovatebot closes pull requests opened by Dependabot (especially if Renovate sends a PR that fixes the same thing?) It's worth noting that all the pull requests I'm referring to were opened by Dependabot before I installed Renovate -- will Renovate suppress future dependabot pull requests but still benefit from dependabot data? 

tl:dr how should I handle PRs sent by dependabot? Does Renovate handle closing those pull requests?

**Comment by secustor** [comment, marked as answer]
Renovate ignores Dependabot PRs, if you don't want their PRs anymore you have to simply delete their config file in `.github/dependabot.yml`
